.github/workflows/set_reviewers.yml

```
@@ -0,0 +1,22 @@
+name: Set Reviewers
+
+on:
+  workflow_dispatch:
```

RE: https://github.com/viamrobotics/viam-cpp-sdk/pull/50/files#r1139082301

does this change the situation at all?

```suggestion
  workflow_dispatch:
  workflow_call:
    secrets:
        GIT_ACCESS_TOKEN:
          required: true
```

---

does not, sadly :(